src/agentscope/models/openai_model.py

```diff
@@ -110,12 +120,14 @@ def _metric(self, metric_name: str) -> str:
         Returns:
             `str`: Metric name of this wrapper.
         """
-        return get_full_name(name=metric_name, prefix=self.model_name)
+        return get_full_name(name=metric_name, prefix=self.model)
 
 
 class OpenAIChatWrapper(OpenAIWrapper):
     """The model wrapper for OpenAI's chat API."""
 
+    alias: str = "openai"
```

Comment:
The similar issue for `model_id`, since we use `model_type` in configuration already. Correspondingly, this cls attribute should be `model_type`.